mobile/android/build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
-        classpath 'com.google.gms:google-services:4.2.0'
+        classpath 'com.android.tools.build:gradle:3.6.3'
```

Comment:
Are you able to get this to build with `3.5.0`? This later version has introduced an NDK build error for some users.

Reply:
yes, without any error